code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -307,19 +307,6 @@
 		return 2
 	return 0
 
-
-/obj/item/storage/fancy/cigar/verb/quick_open_close(mob/user)
-	set name = "Close cigar case"
-	set category = "Object"
-	set src in view(1)
```

Comment:
Would `view(0)` work here?

Reply:
Problem is, it thinks that every mob is mob/user. No idea why. Same issue in hardcases code.

Reply:
What do you mean every mob is mob/user? 🤔

Reply:
![image](https://user-images.githubusercontent.com/49622619/145628034-1b28aa03-904c-4d3f-bec9-e774bd66a619.png)
I mean THIS.
![image](https://user-images.githubusercontent.com/49622619/145628048-0157c446-47bb-4d43-9418-828d1076018c.png)